XRTK.SDK/Packages/com.xrtk.sdk/DefaultProfiles/Input/ControllerMappings/DefaultLuminControllerMappingProfile.asset

```diff
@@ -13,7 +13,6 @@ MonoBehaviour:
   m_Name: DefaultLuminControllerMappingProfile
   m_EditorClassIdentifier: 
   isCustomProfile: 0
-  parentProfile: {fileID: 11400000, guid: f83e1d143c5a46dba5e70f497a7a996d, type: 2}
```

Comment:
Was this breaking of parenting intentional?

Reply:
Yes, this property is no longer serialized.